test/functional/test-resources.js

```diff
@@ -516,6 +516,113 @@ describe('Resources schedulePreload', () => {
 });
 
 
+describe('Resources scheduleUnlayout', () => {
```

Comment:
You can reuse most of the existing scheduleLayout test code. See https://github.com/ampproject/amphtml/pull/4137/files for how I am sharing code between scheduleResume and schedulePause tests.